src/components/Feed.tsx

```diff
@@ -1,13 +1,31 @@
-import React from "react";
+import React, { useState } from "react";
 import NewPost from "./NewPost";
 import PostList from "./PostList";
+import { Button } from "antd";
+import { useAppSelector } from "../redux/hooks";
 
 const Feed: React.FC = () => {
+  const profile = useAppSelector((state) => state.user.profile);
+  const [openCreatePost, setCreatePost] = useState<boolean>(false);
```

Comment:
`setCreatePost` is a little misleading since all this is doing is opening the modal. Perhaps, isModalOpen?